src/meetingsession/MeetingSessionConfiguration.ts

```diff
@@ -53,6 +53,11 @@ export default class MeetingSessionConfiguration {
    */
   connectionHealthPolicyConfiguration: ConnectionHealthPolicyConfiguration = new ConnectionHealthPolicyConfiguration();
 
+  /**
+   * Feature flag to enable WebAudio processing
+   */
+  enableWebAudio: boolean = true;
```

Comment:
I think we should set this to `false` by default so that it's opt-in rather than opt-out.